### PR TITLE
fix clear copyset creating flag when create copyset success

### PR DESCRIPTION
**Pull request by cw123**
### What problem does this PR solve?

Issue Number: #1382

Problem Summary: Copyset is not automatically migrated after stopping metaserver

### What is changed and how it works?

What's Changed: fix clear copyset creating flag when create copyset success

When copyset created success, it should clear the copyset creating flag. But the code miss this in code.

### CODE REVIEW

**Comment by cw123** [comment]
recheck

**Comment by cw123** [comment]
recheck

**Comment by YunhuiChen** [comment]
recheck

**Review by Wine93 — CHANGES_REQUESTED** [review]
LGTM!

**Review by Wine93 — APPROVED** [review]
LGTM!

File: curvefs/src/mds/topology/topology_manager.cpp [hunk 1]
```diff
@@ -866,6 +866,7 @@ TopoStatusCode TopologyManager::CreateCopyset(
         return ret;
     }
 
+    ClearCopysetCreating(copyset.poolId, copyset.copysetId);
```
Comment (wu-hanqing) [inline curvefs/src/mds/topology/topology_manager.cpp:869]: it's better to move this line of code before 856 and remove line 865, so you don't have to repeat it.

and should you add some unit test?

Comment (cw123) [reply curvefs/src/mds/topology/topology_manager.cpp:869]: 虽然都是去掉creating的标记，但是清理的时机是不一样的。如果都挪到856行，和原意还是有区别的。

Although the marking of creating is removed, the timing of cleaning is different. If it is moved to line 856, it is still different from the original intention.

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**